src/tests/ftest/fault_injection/ec.yaml

```diff
@@ -0,0 +1,91 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+    - server-E
+    - server-F
+    - server-G
+  test_clients:
+    - server-H
+timeout: 300
+setup:
+  start_agents_once: False
+  start_servers_once: False
+server_config:
+  engines_per_host: 2
+  name: daos_server
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      bdev_class: nvme
+      bdev_list: ["aaaa:aa:aa.a"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem0"]
+      scm_mount: /mnt/daos0
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      bdev_class: nvme
+      bdev_list: ["bbbb:bb:bb.b"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem1"]
+      scm_mount: /mnt/daos1
+pool:
+    name: daos_server
+    control_method: dmg
+    scm_size: 50%
+    nvme_size: 93%
+    properties: ec_cell_sz:64KiB
+container:
+    type: POSIX
+    control_method: daos
+    properties: cksum:crc16,cksum_size:16384,srv_cksum:on,rf:2
+ior:
+  api: "DFS"
+  client_processes:
+    np: 16
+  dfs_destroy: False
+  iorflags:
+      flags: "-w -r -R -G 1 -k -vv"
+  test_file: /testFile
+  repetitions: 2
+  transfer_size: 4M
+  block_size: 128M
+  objectclass:
+      dfs_oclass:
+      - "EC_2P2GX"
+      - "EC_4P2GX"
+      - "EC_8P2GX"
+fio:
+  names:
+    - global
+    - test
+  global:
+    directory: "/tmp/daos_dfuse"
+    thread: 1
+    group_reporting: 1
+    direct: 1
+    verify: 'crc64'
+    iodepth: 16
+    blocksize: '1M'
+    size: '1G'
+    rw: 'rw'
+  test:
+    numjobs: 1
+dfuse:
+  mount_dir: "/tmp/daos_dfuse"
+faults:
+    fault_list:
+      - DAOS_CSUM_CORRUPT_FETCH
+      - DAOS_DTX_LOST_RPC_REQUEST
+      - DAOS_DTX_LOST_RPC_REPLY
+      - DAOS_SHARD_OBJ_RW_DROP_REPLY
```

Comment:
Do we need to inject all these faults? I would do step by step when adding fault injection to avoid unwanted behaviour.